releases/netcode/1-0-0.md

```diff
@@ -14,50 +14,99 @@ This release contains features, updates, bug fixes, and refactoring for the  Net
 | Netcode for GameObjects | 1.0.0| Release | June 27, 2022 | 2020.3 and later |
 | Netcode for GameObjects | 1.0.1| Release | August 25, 2022 | 2020.3 and later |
 | Netcode for GameObjects | 1.0.2| Release | September 13, 2022 | 2020.3 and later |
+| Netcode for GameObjects | 1.1.0| Release | October 10, 2022 | 2020.3 and later |
 
 :::note
-Netcode for GameObjects supports Windows, MacOS, Ubuntu 20.4 LTS and Ubuntu 18.04 LTS versions of Unity Editor and Player
+Netcode for GameObjects supports Windows, MacOS, Ubuntu 20.4 LTS, and Ubuntu 18.04 LTS versions of Unity Editor and Player
 :::
 
-## [1.0.2] - 2022-09-13
+## [1.1.0] - 2022-10-20
 
-### Fixed
+### Added
 
-* Fixed issue where `NetworkTransform` was not honoring the `InLocalSpace` property on the authority side during `OnNetworkSpawn`. ([#2170](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2170))
-* Fixed issue where `NetworkTransform` was not ending extrapolation for the previous state causing non-authoritative instances to become out of sync. ([#2170](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2170))
-* Fixed issue where `NetworkTransform` was not continuing to interpolate for the remainder of the associated tick period. ([#2170](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2170))
-* Fixed issue during `NetworkTransform.OnNetworkSpawn` for non-authoritative instances where it was initializing interpolators with the replicated network state, which now only contains the transform deltas that occurred during a network tick and not the entire transform state. ([#2170](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/2170))
+* Added `NetworkManager.IsApproved` flag that is set to `true` if a client is approved. (#2261)
```

Comment:
I think we need the Github Links to so it appears as a link to on the Docs website.

Reply:
Done 🙂